src/locales/en.json

```diff
@@ -48,6 +48,7 @@
   "error_api_username_reserved": "Username reserved",
   "error_api_username_used": "This username is already in use",
   "error_api_wait": "Please wait at least one minute between retries",
+  "error_api_actionlimit": "You are doing that too much, please try again tomorrow",
```

Comment:
The key is missing in other languages.

Reply:
That should not block this pr (assuming that we fall back to English if another language is not found, right?)

Can you create an issue about adding the translation later?

Reply:
If I'm not mistaken react-intl we'll use the key of the message if it's not existing in the locale file

Reply:
That's not good, there should be a fallback locale set